### DISCUSSION TITLE
InnoDB가 여러 방면에서 좋아 보이는데, InnoDB가 아닌 MyISAM을 쓰는 경우도 있을까?

### DISCUSSION THREAD

**Discussion by CODe5753** [Well-Founded-Confidence/RealMySQL#15] [Q&A]
4.1장을 전체적으로 보면서 MyISAM과 InnoDB의 차이점을 약간 알게 되었습니다.
근데 문득 궁금한건 InnoDB가 보편적으로 사용되고 있고, MyISAM은 뭔가 레거시 같은 느낌이 들더라구요.
보통 이 경우 MyISAM은 완전히 은퇴(?)를 하면서 잘 언급되지 않을텐데,
책 내용 전반적으로 MyISAM과 비교되는 이유가 궁금했습니다

MyISAM의 비중이 큰 이유와 실제로 MyISAM만을 써야하는 케이스가 있을까요?

**Comment by menthamin** [comment]
MyISAM: 더 빠르다.
- 엔진이 가볍다
- 읽기 전용인 경우
- Select 위주인 경우

InnoDB: 트랜잭션
- 엔진이 보다 무겁다.
- Default로 InnoDB 엔진을 사용하고 있다. (MySQL 5.5 >= )

**Comment by CODe5753** [comment]
Default로 InnoDB 엔진을 사용하고 있는 것으로보아,
특별히 읽기 기능*만* 쓰는 경우가 아니라면 차츰 대체될 것으로 예상됩니다.
